voice/bxml/verbs/record.md

```diff
@@ -9,7 +9,7 @@ The Record verb allows a segment of audio to be recorded during a call. At the e
 | recordCompleteMethod         | (optional) The HTTP method to use for the request to `recordCompleteUrl`. GET or POST. Default value is POST.                                                                                                          |
 | recordingAvailableUrl        | (optional) URL to send the [Recording Available](../callbacks/recordingAvailable.md) event to once it has been processed. Does not accept BXML.                                                                        |
 | recordingAvailableMethod     | (optional) The HTTP method to use for the request to `recordingAvailableUrl`. GET or POST. Default value is POST.                                                                                                      |
-| transcribe                   | (optional) A boolean value to indicate that recording should be transcribed. Default is `false`.                                                                                                                       |
+| transcribe                   | (optional) A boolean value to indicate that recording should be transcribed. Transcription can succeed only for recordings of length greater than 500 milliseconds and less than 4 hours. Default is `false`.                                                                                                                       |
```

Comment:
I would say `can only be performed for recordings`. As written it kind of implies failure, whereas it's really just a constraint.

Reply:
well, you can try a transcription on any recording and it will fail if less than or greater than these constraints. we don't block transcription, we start the process in any situation and it either fails or succeeds. so i think my wording is accurate. do you still disagree?

Reply:
Oh we don't do checks on the recording if you request a transcription via the API? Seems like we should... :thinking:

Reply:
yeah, i followed what was currently in place for the "too long" recordings, where the error comes as a callback, not as an immediate validation error.

Reply:
so it goes to VMS and comes back, it does not go to the actual transcription service